app/src/pages/common/numerableBlock/numerableBlock.jsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2025 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import classNames from 'classnames/bind';
+import React from 'react';
+
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { intlMessageType } from 'common/constants/localization';
+import Parser from 'html-react-parser';
+import styles from './numerableBlock.scss';
```

Comment:
Let's at least keep this order

- third-party imports
- local aliased imports (e.g. `common/...`)
- local relative imports (e.g. `../file`)
- styles
```suggestion
import React from 'react';
import { useIntl } from 'react-intl';
import PropTypes from 'prop-types';
import classNames from 'classnames/bind';
import Parser from 'html-react-parser';
import { intlMessageType } from 'common/constants/localization';
import styles from './numerableBlock.scss';
```

Reply:
Done.